packages/hooks/src/useCollectionQuery/useCollectionQuery.ts

```diff
@@ -296,9 +298,11 @@ function subscribeToMoreHandler<TQuery, TSubscription>(
     );
   }
 
+  const totalCount = outputCollection.totalCount != undefined ? { totalCount: outputCollection.totalCount + 1} : {};
+
   Object.assign(outputCollection, {
-    pageInfo: cloneDeep(outputCollection.pageInfo),
-    totalCount: outputCollection.totalCount + 1,
+    ...{ pageInfo: cloneDeep(outputCollection.pageInfo) },
+    ...totalCount
```

Comment:
Why are you putting pageInfo into an object then expanding the object? Why not define it directly in the parent object. Similarily does totalCount need to be an object that gets expanded? Could you just store it as a number? 

would this work?
`
const totalCount = outputCollection.totalCount != undefined ?  totalCount: outputCollection.totalCount + 1 :  undefined;
Object.assign(outputCollection, {
    pageInfo: cloneDeep(outputCollection.pageInfo) ,
    totalCount
})
`

Reply:
I don't think this would work, because you would need to wrap `{ totalCount: outputCollection.totalCount + 1 }` no?
Then we would end up with: 
```
Object.assign(outputCollection, {
pageInfo: cloneDeep(outputCollection.pageInfo) ,
{ totalCount: 5 }
})
```
In my solution I am just merging 2 hashes.

Reply:
Sorry messed up my example I meant for const totalCount to be a number not an object.

Reply:
Something like this https://jsfiddle.net/pcmjqsn9/1/

Reply:
Oh but we don't always want totalCount to be present! Unless I am missing something about your implementation. Want to chat?

Reply:
That's fair. I figured it didn't matter if the key was added to the object if the value of the property was undefined regardless. It makes sense if you want to avoid having the key on the object at all.